Paws4Me/Paws4Me/Extensions/UIColorExtension.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIColorExtension.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/09.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+    static var myAppPurple: UIColor {
+        return UIColor(named: "primaryPurple") ?? UIColor.clear
```

Comment:
Watch out for defaults as this clear means your back button could be hidden aswell as title investigate further

Reply:
That is very true. Thank you